frontend/src/components/Layout.tsx

```diff
@@ -18,7 +19,7 @@ export default function Layout({ children, locale }: Props) {
   const t = useTranslations();
 
   return (
-    <UserProvider>
+    <UserProvider featureFlagDefaults={defaultFeatureFlags}>
```

Comment:
similar to what we discussed in the middleware, I think this also needs to include env var related flags